lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -4858,16 +4859,61 @@ TypeChecker::findWitnessedObjCRequirements(const ValueDecl *witness,
 
       // Skip types.
       if (isa<TypeDecl>(req)) continue;
+      
+      // Dig out the conformance.
+      Optional<ProtocolConformance *> conformance;
+      if (!conformance.hasValue()) {
```

Comment:
Ugh, that was supposed to be declared outside of the for-each loop. I'll fix on master. Thank you!